src/pcre2_auto_possess.c

```diff
@@ -245,7 +245,7 @@ switch(ptype)
     default:
     return (PRIV(ucp_gentype)[prop->chartype] == ucp_Z) == negated;
     }
-  PCRE2_UNREACHABLE(); /* Control never reaches here */
+  PCRE2_UNREACHABLE(0); /* Control never reaches here */
```

Comment:
this an the next are just leftover from v2, as the PCRE2_UNREACHABLE() doesn't have a parameter anymore and the one that is meant to be used in places were "fall back" code should be kept in non debug build use a different macro instead as suggested by Zoltan